components/Footer.js

```diff
@@ -0,0 +1,206 @@
+import NavLink from './NavLink'
+
+/**
+ * Footer for the main layout.
+ * Derived from https://codepen.io/tutsplus/pen/yWrEgW
+ */
+export default function Footer (props) {
+  return (
+    <footer>
+      <section className='ft-main'>
+        <div className='ft-main-item'>
+          <h2 className='ft-title'>About</h2>
+          <ul>
+            <li>
+              <NavLink
+                // FIXME: Update content at link
+                href='/about'>
+                About this app
+              </NavLink>
+            </li>
+            <li>
+              <NavLink
+                // FIXME: Update content at link
+                href='/faq'>
+                FAQ
+              </NavLink>
+            </li>
+            <li>
+              <NavLink href='https://fdot-rmce.statuspage.io/' external>
+                Status
+              </NavLink>
+            </li>
+          </ul>
+        </div>
+        <div className='ft-main-item'>
+          <h2 className='ft-title'>Resources</h2>
+          <ul>
+            <li>
+              <NavLink
+                // FIXME: Update link
+                href='https://fdot-otp.ibi-transit.com/'
+                external
+              >
+                View Trip Planner
+              </NavLink>
+            </li>
+            <li>
+              <NavLink
+                // FIXME: Update link
+                href='#'
+                external
+              >
+                API documentation
+              </NavLink>
+            </li>
+          </ul>
+        </div>
+        <div className='ft-main-item'>
+          <h2 className='ft-title'>Contact</h2>
+          <ul>
+            <li>
+              <NavLink
+                // FIXME: Update link
+                href='/'>
+                Help
+              </NavLink>
+            </li>
+            <li>
+              <NavLink
+                // FIXME: Update link
+                href='/'>
+                Request access to API
+              </NavLink>
+            </li>
+          </ul>
+        </div>
+        {/* FIXME: Remove subscribe? */}
```

Comment:
Yes, remove

Reply:
It is nice UI for the general public, but I would remove too.